lib/app_state/lazy_load_success.dart

```diff
@@ -1,16 +1,14 @@
 import 'package:fluffychat/app_state/success.dart';
 
 abstract class LazyLoadSuccess<T> extends Success {
```

Comment:
IMO, we dont need to keep it? WDYT? @linagora/twake

Reply:
no need anymore